### DISCUSSION TITLE
奇怪的事情又多了，端口ip阻断都没事 就是连不上

### DISCUSSION THREAD

**Discussion by 23250789** [v2fly/v2ray-core#2400] [General]
之前没套cf，前两天开始突然连接不上了，接着就是开一个端口，连接1分钟不到，然后真延迟-1。
移动访问的时候跟老爷子迈不开腿似的，连宝塔都进不去，起初我以为是墙了ip，但是ping了之后没问题
![image](https://user-images.githubusercontent.com/70897251/224565460-2338e1a6-007a-4323-996e-400c5acc97ae.png)
然后我又去查端口，结果端口也没被封，查了五花八门各种小众端口都没事。
![image](https://user-images.githubusercontent.com/70897251/224565571-7f29f4db-f490-4022-bcf4-2722b11432cd.png)
现在我又去查是不是阻断了，结果也bing并没有
![image](https://user-images.githubusercontent.com/70897251/224565717-5bba2b39-fc59-4bf6-8fa6-1ab512c8cf73.png)
现在我奇怪了，这到底是咋个原因就死活上不去了呢。
![image](https://user-images.githubusercontent.com/70897251/224565773-1523d2b5-d338-47f2-8df3-436e97250fea.png)

**Comment by 23250789** [comment]
各位大佬 求解

**Comment by nursery01** [comment]
154.9.89.227:443  可以正常收到資料
~~結論是你配置寫錯了~~
-------------------update---------------------------
我可能是錯的，有用戶報告瀏覽器可以訪問443，但是v2ray就是連不上，可能是client指紋問題或者是alpn問題

**Reply by 23250789** [reply]
比如说？我都是直接xui生成的...
